docs/cloud/grafana.md

```diff
@@ -0,0 +1,29 @@
+# Grafana
+
+GreptimeCloud service can be configured as a [Grafana data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+## Prometheus
+
+Click the `Add data source` button and select `Prometheus` as the type.
+
+Fill in the following information in `HTTP`:
+
+- Prometheus server URL: `https://<host>/v1/prometheus/read?db=<dbname>`
```

Comment:
@sunng87 @yuanbohan I'm not sure this link is ok.

Reply:
this is link for remote read, let me check for correct url

Reply:
This SHOULD be the prometheus endpoint, not remote read. The original prometheus documentation is [querying](https://prometheus.io/docs/prometheus/latest/querying/api), and I know we are doing some refactoring job in order to merge 4000 and 4004 ports

Reply:
prometheus url:

https://<host>/v1/prometheus/?db=<dbname>